site/learn/expert-advice/identity-basics/slow-migration.md

```diff
@@ -0,0 +1,252 @@
+* This approach takes longer to complete, leaving you running multiple systems for a longer period of time.
+* You'll have downtime for each user segment, which requires additional off hours work or coordination.
+
+## How does a slow migration of auth data work?
+
+At a high level, a slow migration happens in four phases. Each user proceeds through the phases independently of any other.  Here's how the data flows before any changes are made: 
+
+{% plantuml source: _diagrams/learn/expert-advice/identity-basics/slow-migration/auth-before-migration.plantuml, alt: "The user auth process before migration." %}
+
+Then you'd stand up the new system, connect it to the old system, and route all authentication requests to the new system. When the new auth system is in front of the old auth system, the old system consulted the first time a user authenticates:
+
+{% plantuml source: _diagrams/learn/expert-advice/identity-basics/slow-migration/auth-during-migration-first-auth.plantuml, alt: "The auth process during an initial authentication in a slow migration." %}
+
+For subsequent logins, the user's data has been migrated. There's no need to consult the old auth system. However, it continues to run because there are other users who have not yet performed an initial authentication.
+
+{% plantuml source: _diagrams/learn/expert-advice/identity-basics/slow-migration/auth-during-migration-second-auth.plantuml, alt: "The auth process during a subsequent authentication in a slow migration." %}
+
+After a period of time, most or all user data has been migrated. There's no need to consult the old auth system. It can be safely shut down.
+
+{% plantuml source: _diagrams/learn/expert-advice/identity-basics/slow-migration/auth-after-migration.plantuml, alt: "The auth process after migration is completed." %}
+
+Slow migration is similar to the [strangler pattern](https://martinfowler.com/bliki/StranglerFigApplication.html), first documented by Martin Fowler. Let's walk through the steps to successfully undertake a phased migration.
+
+## Plan and map your old auth system to your new one
+
+The first step to any successful data migration is planning. A slow migration is no different. You need to know where all your data sources are, how to connect to them, and what the data looks like. 
+
+Consider edge cases. What fields are required and optional in the old system? What about the new system? 
+
+Is there a clean mapping between the data fields? The answer is almost certainly no, so think about how you are going to handle irregularities.
+
+Let's examine a trivial example. Suppose an old auth system has a user data model with these attribute names and data types:
+
+* fname - string
+* lname - string
+* birthdate - string
+* phone num - string
+
+Assume the new system has a user object with these attributes and data types:
+
+* first name - string
```

Comment:
I'd use either snake, camel or Pascal case here rather than spaces. It will be more approachable to engineers.